.eslintrc

```diff
@@ -18,7 +18,6 @@
         "import/default": "error",
         "import/no-absolute-path": "error",
         "import/no-self-import": "error",
-        "import/no-cycle": "error",
```

Comment:
I had to remove this as Flow imports were now being detected and erroring out (as we had cycles).